### ISSUE TITLE
Feature request: Keyboard modifiers for Flexi

### ISSUE THREAD

**Issue by agrover** [git-moss/DrivenByMoss#388]
Background: I have a Midi Fighter Twister (16 combination encoders/buttons) that I have fully configured using Flexi to change 16 different values and also as 16 different buttons. But I'd like MORE -- access to more slot actions on the same hardware.

Request: I'd like to be able to define a selected slot to not trigger from MIDI, but trigger from MIDI plus [keyboard modkey]. For example, I have MIDI channel 1 CC 0 control Track 1: Set Volume, I'd like to define another slot for Ctrl+MIDI 1 CC0 to control Transport: Set Tempo.

Alternatively, if that's impossible, defining triggers be combined with Global: Shift button would also be a big help -- redefine one of my existing buttons to Global: Shift, and then get access to 15 more slots by pressing/twisting other encoders with that button held.

Basically, watching the Faderfox EC4 video got me jealous I couldn't get so many combinations via Flexi :-) Thanks for everything, btw!

**Comment by git-moss** [comment]
Will be in the next update.
